base_frame: "base_link"
tool_frame: "flange"
test_links: [wrist_3_link, forearm_link, flange, wrist_1_link]

# For new simulations
minimum_distance: 0.25 
reaction_time: 0.15 
human_velocity: 1.6
maximum_cartesian_acceleration: 2.5

time_remove_old_objects: 0.1

#Information to publish obstacles
publish_obstacles: false
sphere_radius: 0.15
sphere_rgba: [1.0,1.0,1.0,0.2]
